airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -202,34 +204,52 @@ def job_completion(self, job_name: str, run_id: str, verbose: bool = False) -> D
         failed_states = ['FAILED', 'TIMEOUT']
         finished_states = ['SUCCEEDED', 'STOPPED']
         next_log_token = None
-        job_failed = False
+        glue_client = self.get_conn()
+        cw_logs = AwsLogsHook()
```

Comment:
Without any parameters to the hook it will use default boto3 strategy, and `cw_logs.get_conn()` turned into `boto3.client("logs")`.

```suggestion
        cw_logs = AwsLogsHook(aws_conn_id=self.aws_conn_id, region_name=self.region_name)
```

But it really nice that you move code related to AWS CloudWatch to the `AwsLogsHook`

Reply:
Thanks, @Taragolis  I have made changes as per the suggestion.